### ISSUE TITLE
Crelay crash

### ISSUE THREAD

**Issue by Sztoppolino** [ondrej1024/crelay#36]
Hi!

I have a problem. I use crelay on my OpenWRT router. It succesfully installed and it control my 1-CH relay card properly. 

Unfortunately, crelay always crash after undefinied time (~1 minute - ~1 day). Sometimes a fast router-reboot is enough, but usually I have to re-install the package.

This issue is very weird, because I can't find any reasons. It happens randomly. When I use (switch on/off) the relay, suddenly the web UI disappear, and my web broswer can't estabilish the connection with crelay anymore. Moreover, PiRelay neither. It sends me an error message:

"Pi (server) not detected on network"

Do you have any ideas (or solutions?)

Thank you in advance! ;)

**Comment by ondrej1024** [comment]
This is probably the same issue which was reported [here](https://github.com/ondrej1024/crelay/issues/28). Please take a look. There is no proper solution for this at the moment. :disappointed:

**Comment by Sztoppolino** [comment]
Hi!
Thank you for your answer. I examined again this issue, and it seems I have the same problem. When I swich ON my relay, crelay can send the command but thereafter (-> try to turn off) my PC lose the connection and crelay crash. I noticed that this problem always occur when I try to switch OFF my relay.

So I tried to confirm [this suspected](https://github.com/ondrej1024/crelay/issues/28) issue. I disconnected the relay module from 230v AC, and I switched on/off the relay for 5 minutes. Crelay didn't crash.

Thereafter, I connected to 230v AC again, and crelay crashed after 30 seconds.

So I agree with you, the 230V can be a good reason. That makes sense.

-------------------------------------------------------------------------------------------------------------------

Did you try ferrit choke? Can I avoid this problem, if I use it?
